packages/core/src/render3/interfaces/definition.ts

```diff
@@ -323,7 +323,8 @@ export interface ComponentDef<T> extends DirectiveDef<T> {
   tView: TView|null;
 
   /**
-   * A function added by the {@see ɵɵStandaloneFeature} and used by the framework to create standalone injectors.
+   * A function added by the {@link ɵɵStandaloneFeature} and used by the framework to create
```

Comment:
That's an internal doc, I think we can just drop the tag altogether:
```suggestion
   * A function added by the `ɵɵStandaloneFeature` and used by the framework to create
```

Reply:
Marked the PR as ready for merge, my comment is really minor, please ignore :)

Reply:
`ComponentDef` is not a public API anyway, so these do not end up in docs. Having the `@link` here helps people working on the source code as the link shows up in intellisense, so I think keeping it a good idea.